README.md

```diff
@@ -178,6 +178,7 @@ The CB-Geo MPM code uses a `JSON` file for input configuration. To run the mpm c
 For example:
 
 ```
+export OMP_SCHEDULE="static, 4"
```

Comment:
@kks32 I got a short question: here, the chunk_size is set to be 4, whereas the `-p 8` option is used. In this case, how many threads will be used to run the `parallel for`, with and without the `schedule(runtime)`?

Reply:
chunk size and number of threads are different. http://jakascorner.com/blog/2016/06/omp-for-scheduling.html `-p 8` sets the number of threads, not the chunk size.

Reply:
I think it would still be 8 threads as there will be block cycling scheduling or like "round robin".